orcid-message-listener/src/main/resources/log4j.xml

```diff
@@ -46,7 +46,7 @@
 	<!-- Root Logger -->
 	<root>
 		<level value="INFO" />
-		<!-- <appender-ref ref="CONSOLE" /> -->
+		<appender-ref ref="CONSOLE" />
```

Comment:
Do we really want to send all login to the console as well?

Reply:
Nope, that was from my debugging.  I'll turn it off.